src/api/activiti-rest-api/docs/RelatedProcessTask.md

```diff
@@ -0,0 +1,7 @@
+# RelatedProcessTask
+
+## Properties
+| Name | Type | Description | Notes |
+| ------------ | ------------- | ------------- | ------------- |
+| **processId** | **string** |  | [optional] [default to null] |
```

Comment:
for both properties - should we say default to null or default to undefined?

Reply:
Changed to undefined